### ISSUE TITLE
Remembering and restoring glPolygonMode after rendering UI frame

### ISSUE THREAD

**Issue by laralex** [pyimgui/pyimgui#295]
Currently in `integrations/opengl.py` the code restores the modified OpenGL state after drawing the UI frame:
```
gl.glUseProgram(last_program)
gl.glActiveTexture(last_active_texture)
gl.glBindTexture(gl.GL_TEXTURE_2D, last_texture)
gl.glBindVertexArray(last_vertex_array)
gl.glBindBuffer(gl.GL_ARRAY_BUFFER, last_array_buffer)
gl.glBindBuffer(gl.GL_ELEMENT_ARRAY_BUFFER, last_element_array_buffer)
gl.glBlendEquationSeparate(last_blend_equation_rgb, last_blend_equation_alpha)
gl.glBlendFunc(last_blend_src, last_blend_dst)
#... goes further
```
One thing is doesn't set and doesn't restore is `glPolygonMode`, thus the GUI may be drawn with a custom mode like `GL_LINE`, `GL_POINT`.
![image](https://user-images.githubusercontent.com/20727983/191367373-d5eb9a5a-022f-4d6e-adff-c00655920823.png)
On one hand it can be considered a desired behavior (who knows how users want to draw the UI), however a tiny bit annoying when it's not needed (requiring to restore it by myself)

**Comment by learn-more** [comment]
This seems like a bug to me, so it should be fixed.
Thanks for reporting this!

**Comment by KinoxKlark** [comment]
This should be fix with #296, thus I close the issue
